rust/cargo-psibase/src/config.rs

```diff
@@ -0,0 +1,70 @@
+use anyhow::{anyhow, Error};
+use serde::{Deserialize, Serialize};
+use std::collections::HashMap;
+use std::path::{Path, PathBuf};
+use url::Url;
+
+#[derive(Serialize, Deserialize, Default)]
+pub struct PsibaseConfig {
+    pub hosts: HashMap<String, String>,
+}
+
+fn psibase_config_path() -> Result<PathBuf, Error> {
+    let home = std::env::var("HOME")?;
+    let path = Path::new(&home).join(".psibase.toml");
```

Comment:
I'd still prefer this to be parsed by `psibase` instead, but if it's specific to `cargo psibase`, then it should not take such a generic name (though admittedly, `.toml` kind of indicates an association with cargo).